android/beagle/src/main/java/br/com/zup/beagle/android/imagedownloader/BeagleImageDownloader.kt

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2020 ZUP IT SERVICOS EM TECNOLOGIA E INOVACAO SA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package br.com.zup.beagle.android.imagedownloader
+
+import android.widget.ImageView
+
+interface BeagleImageDownloader {
+    fun download(url: String, imageView: ImageView)
```

Comment:
What do you think about keeping the `RootView` in the parameter? like I think if you need maybe another developer can need too
```suggestion
    fun download(url: String, imageView: ImageView, rootView: RootView)
```

Reply:
Done!